config/toggles.js

```diff
@@ -60,6 +60,8 @@ const longTermToggles = {
   enableKueStatsActivityLogs:
     environmentVariableTrue(pe, 'ENABLE_KUESTATS_ACTIVITY_LOGS'),
 
+  enablePubStatsLogs: environmentVariableTrue(pe, 'ENABLE_PUB_STATS_LOGS'),
```

Comment:
Can you make one line comment over here about this toggle variable?

Reply:
done